### PR TITLE
feature/sanitizing Social data

### PR DESCRIPTION
**Pull request by MAlngaawy**

### CODE REVIEW

File: src/store/actions/social.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
 import { ACTION_TYPES } from '../actionTypes';
 
+const dataSanitization = data => {
+  if (data) {
+    const sanitizedSocial = data.map(({ _id, url, fontawesome_icons }) => {
+      return {
+         _id, url, fontawesome_icons 
+      };
+    });
+    return sanitizedSocial;
```
Comment (Nilegfx) [inline src/store/actions/social.js:10]: no need to declare a variable here, just return the result of the `map`

File: src/store/actions/social.js [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+  } else {
+    return null;
```
Comment (Nilegfx) [inline src/store/actions/social.js:12]: - no need for the else statment
- don't change the data type (return the data whatever it is)

File: src/store/actions/social.js [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
+const dataSanitization = data => {
+  if (data) {
+    return data.map(({ _id, url, fontawesome_icons }) => {
+      return {
+        _id,
+        url,
+        fontawesome_icons
+      };
+    });
+  }
```
Comment (Nilegfx) [inline src/store/actions/social.js:12]: return data

File: src/store/actions/social.js [hunk 4]
```diff
@@ -1,9 +1,22 @@
 import { ACTION_TYPES } from '../actionTypes';
 
+const dataSanitization = data => {
+  if (data) {
+    data = data.map(({ _id, url, fontawesome_icons }) => {
+      return {
+        _id,
```
Comment (elhaw) [inline src/store/actions/social.js:7]: we can use `id` instead of `_id` for consistency.

**Pull request merged**